### PR TITLE
Update thieving data.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

File: data/base-stealing.yaml [hunk 1]
```diff
@@ -5454,9 +5466,11 @@
   acceptable_min: 284
   poorly_max: 291
   poorly_min: 291
+  vpoorly_max: 387
+  vpoorly_min: 387
 - id: '522'
   item: hip pouch
-  item_in: ' catalog'
+  item_in: " catalog"
```
Comment (rpherbig) [inline data/base-stealing.yaml]: This doesn't look right. Do you know anyone in Shard that could double check it?

File: data/base-stealing.yaml [hunk 2]
```diff
@@ -5767,7 +5791,7 @@
   trivial_max: 1750
   trivial_min: 573
 - id: '556'
-  item: ' mesh handguards'
+  item: " mesh handguards"
```
Comment (rpherbig) [inline data/base-stealing.yaml]: Similar comment to above

**Comment by rpherbig** [comment]
Other than the two comments, this looks fine to merge.

**Comment by Sarvatt** [comment]
I figured the change from single to double quotes were harmless and just someone trying to manually steal with different quotes to update the info. Reverted that change.

**Pull request merged**